hooks/tk-multi-publish2/nuke/publish_files.py

```diff
@@ -55,14 +55,14 @@ def _framerange_to_be_published(self, item):
         # Checking with root because _sync_frame_range() will ensure root is up to date with shotgun
         if missing_frames:
             self.logger.warning("Incomplete renders! All the frames are not rendered.")
-            return False
+            nuke.message("WARNING! Incomplete renders. All the frames are not rendered.")
```

Comment:
nuke.warning() will not generate a pop up dialog. It will just put the message in the error console treating it like a warning and could be missed if the console is not open.